packages/lit-dev-content/site/docs/libraries/bundles.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Bundles
+eleventyNavigation:
+  key: Bundles
+  parent: Related libraries
+  order: 4
+---
+
+Lit is available as pre-built, single-file bundles. These bundles are useful for
+situations where you can't use npm or just want to experiment without setting up
+a build process.
+
+<div class="alert alert-warning">
+
+**If you're using npm for client-side dependencies, you should use the [`lit`
+package](https://www.npmjs.com/package/lit) instead of these bundles.** The
+bundles intentionally combine most or all of Lit into a single file. This can
+prevent some build tools from removing the parts of Lit your project doesn't
+use, which means your users may download more code than they strictly need.
+
+</div>
+
+## Download
+
+The bundles can be found on GitHub in [the lit/dist
```

Comment:
Can we make the first thing here the actual link to the thing you should download (maybe one for min and one for al)?

Reply:
Combined organization suggestion based on other feedback:

##Download
 * start with 2 links to most current versions of min and all.
 * describe contents (from variants) and give warning about choosing and using one of them.
 * just say that source maps are included.
 
 ##Usage
  * just show the example usage for each variant and say they are each in their own folder.
  
  ##CDN

Reply:
I think the unversioned link to jsDeliver now covers this.